toolchain/lower/handle.cpp

```diff
@@ -11,6 +11,7 @@
 #include "llvm/IR/Type.h"
 #include "llvm/IR/Value.h"
 #include "llvm/Support/Casting.h"
+#include "toolchain/check/sem_ir_diagnostic_converter.h"
```

Comment:
Is this needed here? I'm not seeing a use added, so maybe this is some leftover intermediate state?

```suggestion
```